src/tenants/treesforjane/RegisterTrees/RegisterTrees/UploadImages.tsx

```diff
@@ -119,7 +111,10 @@ export default function UploadImages({
                   />
                   {/* <div className={styles.uploadedImageOverlay}></div> */}
                   <div className={styles.uploadedImageButtonContainer}>
-                    <button id={'uploadImgDelIcon'} onClick={() => deleteContributionImage(image.id)}>
+                    <button
+                      id={'uploadImgDelIcon'}
+                      onClick={() => setImage(null)}
+                    >
```

Comment:
Create a circle white background or else the delete icon won't be visible for many images

<img width="487" alt="Screenshot 2021-09-16 at 4 38 18 PM" src="https://user-images.githubusercontent.com/25605193/133602018-418abbd5-c8ae-40a0-8919-edf07ffd98a7.png">